azext_iot/central/commands_device.py

```diff
@@ -68,10 +68,16 @@ def registration_info(
     device_id=None,
     token=None,
     central_dns_suffix="azureiotcentral.com",
+    device_status=None,
 ):
-    provider = CentralDeviceProvider(cmd=cmd, app_id=app_id, token=token)
+    provider = CentralDeviceProvider(cmd=cmd, app_id=app_id, token=token,)
```

Comment:
Why is there a `,` at the end?

Reply:
We use an auto-formatter called "Black", that is adding this additional comma at the end

Reply:
Though in this particular case it should probably be removed

Reply:
I would not worry about this one. This is standard `python black` behavior, in this case adding a comma after the last kwarg element. If you remove it now (which is fine), the next change to this module and a re-run of the formatter would re-introduce it.

`python black` is an opinionated formatter with minimal configuration. Idea for using `Black` is to save time and mental energy for more important matters. By using it, you agree to cede control over minutiae of hand-formatting. Also `Black` will check that the reformatted code still produces a valid AST that is equivalent to the original. It's a _relatively_ newer Python tool that has [immense popularity](https://github.com/psf/black/network/dependents?package_id=UGFja2FnZS01MDMzMzg4OQ%3D%3D) in the open source world.